### PR TITLE
chore(trading,governance,explorer): nx migration to latest

### PR DESCRIPTION
**Pull request by mattrussell36**
# Related issues 🔗

N/A

# Description ℹ️

- Update all with nx migrate latest
- Add tailwind ordering plugin for prettier to ensure correct order of classes
-  Upgrade node version to latest vesrion with lts (16 is end of life)
- Fix failing tests

### CODE REVIEW

File: apps/explorer/src/app/components/order-details/amend-order-details.spec.tsx [hunk 1]
```diff
@@ -223,7 +225,8 @@ describe('Amend order details', () => {
     expect(await res.findByText('-7879')).toBeInTheDocument();
   });
 
-  it('Fetches latest version when version is not specified', async () => {
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('Fetches latest version when version is not specified', async () => {
```
Comment (mattrussell36) [inline apps/explorer/src/app/components/order-details/amend-order-details.spec.tsx:229]: @edd couldn't figure out why this wasn't working, if you have a mo can you take a look?

Comment (edd) [reply apps/explorer/src/app/components/order-details/amend-order-details.spec.tsx:229]: Looking...

Comment (edd) [reply apps/explorer/src/app/components/order-details/amend-order-details.spec.tsx]: Fixed and re-enabled. Unsure why this change raised the problem, but mocking an extra GraphQL call fixed the test.

**Review by bwallacee — APPROVED** [review]
LGTM, this fixes the issues I have currently with building the project.
Docker image creation all works correctly here still.

**Review by asiaznik — APPROVED** [review]

**Pull request merged**